src/hackacrow.cr

```diff
@@ -9,20 +9,20 @@ module Hackacrow
     VERSION
   end
 
-  def self.display_help
-    puts "\n"
-    puts "Usage: hackacrow [OPTIONS]"
-    puts "Options:"
-    puts "  -h, --help        Display this help menu"
-    puts "  -c, --check       Check the output of a command"
-    puts "  -a, --argument    Instead of using stdin to test, use args"
-    puts "  -i, --input FILE  Specifies the JSON input file"
-    puts "  -o, --output FILE Specifies the JSON output file"
+  def self.display_help(stdout : IO = STDOUT)
+    stdout.puts "\n"
+    stdout.puts "Usage: hackacrow [OPTIONS]"
+    stdout.puts "Options:"
+    stdout.puts "  -h, --help        Display this help menu"
+    stdout.puts "  -c, --check       Check the output of a command"
+    stdout.puts "  -a, --argument    Instead of using stdin to test, use args"
+    stdout.puts "  -i, --input FILE  Specifies the JSON input file"
+    stdout.puts "  -o, --output FILE Specifies the JSON output file"
   end
 
-  def self.main
+  def self.main(stdout : IO = STDOUT)
```

Comment:
@lanjoni I needed to pass STDOUT as an argument so that we could unit test the flow control.